src/Mod/CAM/Path/Geom.py

```diff
@@ -252,6 +252,8 @@ def xy(point):
 
 
 def speedBetweenPoints(p0, p1, hSpeed, vSpeed):
+    """speedBetweenPoints(p0, p1, hSpeed, vSpeed)
+    TODO: Needs docstring description here."""
```

Comment:
Please suggest a succinct and relevant docstring

Reply:
Or should we lift this linter warning instead?